docs/AddingNewProjects.md

```diff
@@ -0,0 +1,44 @@
+# Adding New Projects To The Repo
+
+Sample PR of final result: https://github.com/dotnet/aspnetcore/pull/41945
+
+## Creating a new project
+1. Create a new folder that will house your `.csproj` and other project related files.
+2. In VS, Add a new `Solution Folder` in the same place as the new folder from the previous step.
+3. Create the project via the VS `Add` menu (select the folder -> right click -> Add -> follow the wizard).
+4. Add the following `ItemGroup` to your newly generated `.csproj`:
```

Comment:
Nit: A brief explainer to outline that this is for interacting with the PublicAPI analyzer?